bootstrap/master.sh

```diff
@@ -25,13 +25,26 @@ fi
 systemctl daemon-reload
 systemctl restart kubelet
 
-echo "Inititializing the master..."
+# execute modprobe on node - workaround for heketi gluster
```

Comment:
Could you explain it. It's enough to have it here as a comment, just for reference.